tests/testthat/test-new_name.R

```diff
@@ -8,7 +8,6 @@ test_that("new_name() works with no error", {
 })
 
 test_that("new_name() creates 'new' with no error ", {
-  skip("First new_name() needs to gain the argument `board`")
```

Comment:
Now that we have a way to pass a `board`, it's time to go back to tackling this test and try make it pass.

Reply:
I've updated the `new_name()` function and changed the arguments that are passed to the function in our tests. This seems to have worked :)

@lindadelacombaz we should now write a test to check that `new_name()` also removes the old pin, right?